pkg/resource/acl/sdk.go

```diff
@@ -164,6 +164,11 @@ func (rm *resourceManager) sdkFind(
 	}
 
 	rm.setStatusDefaults(ko)
+	// custom set output from response
+	ko, err = rm.customDescribeACLsSetOutput(ctx, r, resp, ko)
+	if err != nil {
+		return nil, err
+	}
```

Comment:
I'm not super fan of this function's signature. Can you change it to take the minimum possible of arguments.
You could also return the events instead of the `ko` and do something like
```
ko.Status.Events, err = rm.customDescribeACLsSetOutput(ctx, r, resp)
```

Reply:
Also at this stage you know that, `len(resp.ACL) == 1` so no need to check the length at all inside `customDescribeACLsSetOutput`

Reply:
Yeah, I forgot that `resp` is checked before in `sdkFind`

Reply:
`ko, err = rm.customDescribeACLsSetOutput(ctx, r, resp, ko)` this is generated by `code-generator`. I didn't write it in `templates`, so I cannot change it.

Reply:
If you want to do that way, I can move them to `sdk_read_many_post_set_output.go.tpl` instead of using `set_output_custom`

Reply:
Ohhh i didn't know that a generated signature. Yes i think `sdk_read_many_post_set_output.go.tpl` would be better here